### DISCUSSION TITLE
Android Studio Giraffe | 2022.3.1 Patch 4 Build error

### DISCUSSION THREAD

**Discussion by Govindvishnu** [software-mansion/react-native-reanimated#5649] [Q&A]
Task :app:buildCMakeDebug[arm64-v8a] FAILED C/C++: ninja: error: 'D:/AndroidProjects/baresip-studio-59.4.0/distribution/baresip/lib/arm64-v8a/libbaresip.a', needed by 'D:/AndroidProjects/baresip-studio-59.4.0/app/build/intermediates/cxx/Debug/2w53694s/obj/arm64-v8a/libbaresip.so', missing and no known rule to make it

**Comment by MatiPl01** [comment]
Hey!

Do you still have this issue or have you managed to resolve it?
